### DISCUSSION TITLE
Error message after migrating Wagmi to V2

### DISCUSSION THREAD

**Discussion by Inevitable-Design** [rainbow-me/rainbowkit#1762] [Q&A] [answered]
Hello,
I recently migrated Wagmi to V2 in my Next.js project (page router), what happened is that it started showing me this error (attached), though I can connect to wallets just fine. I'm kinda new with Wagmi and Rainbowkit, so I'm not sure what could be going wrong here.

Any help is greatly appreciated! 😊

Error
```
[missing "en-US.connector_group.installed" translation]
```
attached.1
![image](https://github.com/rainbow-me/rainbowkit/assets/96431359/b8d6b078-2aaa-48ab-90c3-6fb4a1608b3a)

**Comment by 0xmkh** [comment]
@Inevitable-Design Please update to the latest beta version `2.0.0-beta.6`. Run this command npm install `@rainbow-me/rainbowkit@2.0.0-beta.6`

**Reply by 0xmkh** [reply, marked as answer]
@Inevitable-Design We just released RainbowKit v2 so feel free to not use beta anymore and instead update to version `2.0.0`

**Comment by Inevitable-Design** [comment]
Thanks a lot!!
